### PR TITLE
4906: new footer!

### PR DESCRIPTION
**Pull request by mgamini**
tenor-161770696

Signed-off-by: Garrett Amini <kadjar@gmail.com>

https://github.com/habitat-sh/habitat/issues/4906

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by cnunciato** [comment]
When I look at this on mobile, the footer links wrap in a way that overlaps with the social icons:

image

**Comment by cnunciato** [comment]
Also seeing some side-scrolling:

**Comment by cnunciato** [comment]
On the Enterprise page, the Get Started button's label alignment seems off:

image

File: www/source/images/icons/icon-slack-white.svg [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (cnunciato) [inline www/source/images/icons/icon-slack-white.svg]: These social icons are currently being represented with a PNG sprite and accompanying CSS. I like that we're replacing them with SVGs, but could we also remove the dead CSS code and `icons-social` images?

Comment (cnunciato) [reply www/source/images/icons/icon-slack-white.svg]: Yeah, I mean honestly, if it were me, I'd clean it out for sure. It's just good practice not to leave unused stuff around when you're aware of it.

File: www/source/layouts/_footer.slim [hunk 2]
```diff
@@ -1,111 +1,53 @@
...
+          =link_to 'Learn', '/learn'
+        li.footer--sitemap--link
+          =link_to 'Docs', '/docs'
+        li.footer--sitemap--link
+          =link_to 'Community', '/community'
+        li.footer--sitemap--link
+          =link_to 'Enterprise', '/enterprise'
+        li.footer--sitemap--link
+          =link_to 'Blog', '/blog'
+        li.footer--sitemap--link
```
Comment (cnunciato) [inline www/source/layouts/_footer.slim]: This ends up rendering as an empty `a` tag, which throws off the center alignment of the footer items on mobile:

image

Do we need a link to Builder in the footer? If we do, we should probably put the word `Builder` in there.

Comment (mgamini) [reply www/source/layouts/_footer.slim]: done!

**Pull request merged**